ui/packages/consul-ui/tests/integration/serializers/auth-method-test.js

```diff
@@ -47,30 +56,35 @@ module('Integration | Serializer | auth-method', function(hooks) {
       const request = {
         url: `/v1/acl/auth-method/${id}?dc=${dc}${
           typeof nspace !== 'undefined' ? `&ns=${nspace}` : ``
-        }`,
+        }${typeof partition !== 'undefined' ? `&partition=${partition}` : ``}`,
       };
       return get(request.url).then(function(payload) {
         const expected = Object.assign({}, payload, {
           Datacenter: dc,
           [META]: {
             [DC.toLowerCase()]: dc,
             [NSPACE.toLowerCase()]: nspace || undefinedNspace,
+            [PARTITION.toLowerCase()]: partition || undefinedPartition,
           },
           Namespace: payload.Namespace || undefinedNspace,
-          uid: `["${payload.Namespace || undefinedNspace}","${dc}","${id}"]`,
+          Partition: payload.Partition || undefinedPartition,
+          uid: `["${payload.Partition || undefinedPartition}","${payload.Namespace ||
+            undefinedNspace}","${dc}","${id}"]`,
         });
         const actual = serializer.respondForQueryRecord(
           function(cb) {
             const headers = {
               [DC]: dc,
               [NSPACE]: nspace || undefinedNspace,
+              [PARTITION]: partition || undefinedPartition,
             };
             const body = payload;
             return cb(headers, body);
           },
           {
             dc: dc,
             ns: nspace,
+            partition: partition,
```

Comment:
Changes to serializer tests just add `partition`, so again the changes here are done the same for every data model, again and again and again.